Source/Fuse.Navigation/Router.ScriptClass.uno

```diff
@@ -82,22 +82,22 @@ namespace Fuse.Navigation
 			@see fuse/navigation/router/push
 			@see fuse/navigation/router/gotoRelative
 		*/
-		static void PushRelative(Context c, Router r, object[] args)
+		static void PushRelative(Router r, object[] args)
 		{
-			var route = GetRelative(c, r, args);
+			var route = GetRelative(r, args);
 			if (route != null)
 				r.Modify( ModifyRouteHow.Push, route, NavigationGotoMode.Transition, "" );
 		}
 		
-		static RouterPageRoute GetRelative(Context c, Router r, object[] args)
+		static RouterPageRoute GetRelative(Router r, object[] args)
 		{
 			if (args.Length < 1)
 			{
 				Diagnostics.UserError( "Router.gotoRelative(): requires 1+ parameters", r );
 				return null;
 			}
 			
-			var node = c.Wrap(args[0]) as Node;
+			var node = args[0] as Node;
```

Comment:
Is this `Wrap` taken care of implicitly now?

Reply:
Yes, at least in this case. Wrap is about bridging a set of known types between the JS and UI thread. We're nod doing mirroring, which also bridge objects, but with slightly different semantics. It seems the biggest difference is the handling of `Scripting.External` , which I'm not even sure what is... There's some minor details about float, int and uint as well, but in those cases, simple casts in the user-code seems more appropriate.

However, it's worth noting that in this particular case, this change was already a nop. Context.Wrap for "unknown" types is just a nop, and that's what happened here.

Reply:
Nevermind, I just re-read the code after eating some food, and `Scripting.External` is indeed handled in the mirroring also. So I think the semantics are the same already.